assets/app/view/part/base.rb

```diff
@@ -113,6 +113,12 @@ def translate
         "translate(#{x} #{y})"
       end
 
+      def scale
+        s = render_location[:scale] || 1
+
+        "scale(#{s})"
```

Comment:
make this a oneline "scale(#{render_location...})"

Reply:
2b84872